content/20.treatments.md

```diff
@@ -472,13 +482,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
 
 ###### Current Evidence
 
-A recent preprint postulated that an individual’s vitamin D status may significantly affect one’s risk of developing COVID-19.
-This hypothesis was derived from the fact that the current pandemic occurred in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer, the number of cases was low at that time coinciding with higher 25-hydroxyvitamin D concentrations [@doi:10.20944/preprints202003.0235.v2].
+A recent review postulated that an individual’s vitamin D status may significantly affect their risk of developing COVID-19.
+This hypothesis was derived from the fact that the current pandemic emerged in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to a lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer and higher 25-hydroxyvitamin D concentrations would be higher, the number of cases was low [@doi:10.3390/nu12040988].
 The authors suggest that people at risk of developing COVID-19 should increase their vitamin D3 intake to reach 25-hydroxyvitamin D plasma concentrations above 40–60 ng/ml.
 The authors also suggest supplementation of vitamin D to treat infected patients and to prevent infection in hospital staff.
-While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels effect COVID-19 specifically.
-Likewise, it is assumed that COVID-19 may be seasonal, but there are multiple other factors at play that can affect vitamin D levels that need to be considered.
-These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111].   
+While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels affect COVID-19 outside of this geographic/climatic correlation.  
+Likewise, though it is assumed that COVID-19 may be seasonal, multiple other factors that can affect vitamin D levels should also be considered.
+These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111]. 
+As the pandemic evolves, further research has investigated some of the potential links identified in the Grant et al. review [@doi:10.3390/nu12040988] between vitamin D and COVID-19 and sought to shed light on whether there is any  prophylactic and/or therapeutic relationship. 
```

Comment:
Is this the same review? If so, the reference and authors might better be mentioned in the first sentence.